src/app/core/competition/logging/filter-information.ts

```diff
@@ -0,0 +1,50 @@
+import {MediaObjectDescriptor} from "../../../../../openapi/cineast";
+import {ColorLabel} from "../../../shared/model/misc/colorlabel.model";
+import {Tag} from "../../selection/tag.model";
+
+/**
+ * A simple data class for logging
+ */
+export class FilterInformation {
+
+  /**
+   * When set to true, objects who have metadata matching for any of the categories are displayed.
+   */
+  _useOrForMetadataCategoriesFilter = false;
```

Comment:
I highly discourage the usage of underscores, in order to follow coding conventions from the industry (e.g. the [Google Typescript Styleguide](https://google.github.io/styleguide/tsguide.html#naming-style)